src/main/java/com/amida/saraswati/jdbcsink/model/InputIngest.java

```diff
@@ -13,10 +13,10 @@
 	@GeneratedValue(strategy = GenerationType.AUTO)
 	private Long id;
 
-	@JsonProperty("firstName")
+	@JsonProperty("pateintFirstName")
```

Comment:
this should be `patientFirstName`. I fixed this in the processor.

Reply:
Corrected.